packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,183 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea, TextareaProps } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import clsx from 'clsx';
+import React from 'react';
+import { designTokenStory } from './util';
+
+interface TextareaStoryProps extends TextareaProps {
+  focus?: boolean;
+  focusVisible?: boolean;
+  hover?: boolean;
+  children?: any;
```

Comment:
`children: any` is niet de ideale type, de precieze manier is om `PropsWithChildren<TextareaProps>` te gebruiken.

Rich text in een `<textarea>` gebruiken is technisch mogelijk, maar _frowned upon_ (en je kunt `ReactNode` niet assignen aan `defaultValue`). Als je het voor de story wilt beperken, dan zou ik `children?: string;` voorstellen.